mkt/webapps/management/commands/process_addons.py

```diff
@@ -23,41 +18,11 @@
                                               mkt.STATUS_PUBLIC,
                                               mkt.STATUS_APPROVED],
                                   disabled_by_user=False)]},
-    'add_uuids': {'method': add_uuids,
-                  'qs': [Q(guid=None), ~Q(status=mkt.STATUS_DELETED)]},
     'update_supported_locales': {
         'method': update_supported_locales,
         'qs': [Q(disabled_by_user=False,
                  status__in=[mkt.STATUS_PENDING, mkt.STATUS_PUBLIC,
                              mkt.STATUS_APPROVED])]},
-    'dump_apps': {'method': dump_apps,
```

Comment:
You'd think this is used, but actually we use the `export_data` management command, which calls the `dump_apps` task